src/app/util/Request/Request.js

```diff
@@ -180,3 +180,13 @@ export const listenForBroadCast = name => new Promise((resolve) => {
         };
     }
 });
+
+export const debounce = (callback, delay) => {
```

Comment:
Like, this makes sense on scroll, but where did we managed to get such necessity?